lib/screens/app/explorer/overview.dart

```diff
@@ -1,54 +1,195 @@
+import 'package:seeds/providers/notifiers/planted_notifier.dart';
+import 'package:seeds/providers/notifiers/voice_notifier.dart';
 import 'package:seeds/providers/services/navigation_service.dart';
 import 'package:seeds/widgets/main_card.dart';
-// import 'package:seeds/widgets/seeds_button.dart';
+import 'package:shimmer/shimmer.dart';
+import 'package:seeds/utils/string_extension.dart';
 
-class Overview extends StatelessWidget {
+class Overview extends StatefulWidget {
   const Overview({
     Key key,
   }) : super(key: key);
 
-  Widget buildCategory(String title, String iconName, Function onTap) {
-    return MainCard(
-      onPressed: onTap,
-      margin: EdgeInsets.only(bottom: 10),
-      padding: EdgeInsets.all(15),
-      child: Row(
-        children: <Widget>[
-          Container(
-            width: 28,
-            height: 28,
-            margin: EdgeInsets.only(right: 15),
-            child: Image(
-              image: AssetImage(iconName),
+  @override
+  _OverviewState createState() => _OverviewState();
+}
+
+class _OverviewState extends State<Overview> {
+  Widget buildCategory(
+    String title,
+    String subtitle,
+    String iconName,
+    String balanceTitle,
+    String balanceValue,
+    Function onTap,
+  ) {
+    var width = MediaQuery.of(context).size.width;
+
+    return Row(
+      children: [
+        Container(
+          width: width * 0.58,
+          child: InkWell(
+            onTap: onTap,
+            child: MainCard(
+              margin: EdgeInsets.only(bottom: 10),
+              padding: EdgeInsets.all(15),
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Container(
+                      width: 28,
+                      height: 28,
+                      margin: EdgeInsets.only(right: 15),
+                      child: SvgPicture.asset(iconName)),
+                  Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(
+                          title,
+                          style: TextStyle(
+                            fontWeight: FontWeight.w600,
+                            fontSize: 16,
+                          ),
+                        ),
+                        Padding(padding: EdgeInsets.only(top: 4)),
+                        Text(
+                          subtitle,
+                          style: TextStyle(
+                            fontWeight: FontWeight.w300,
+                            fontSize: 12,
+                          ),
+                        )
+                      ]),
+                ],
+              ),
             ),
           ),
-          Flexible(
-              child: Text(title,
-                  style: TextStyle(fontWeight: FontWeight.w600, fontSize: 16)))
-        ],
-      ),
+        ),
+        Padding(padding: EdgeInsets.only(left: 8)),
```

Comment:
It's always better to wrap the child with padding widget instead of add new padding widget.

Reply:
Ok